components/input/RichTextEditor.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import ReactQuill from 'react-quill';
 import Quill from 'quill';
 
+// This style will be lazy loaded by LazyRichTextEditor
```

Comment:
Quick question: 

there were 
```
@import '~react-quill/dist/quill.core.css';
@import '~react-quill/dist/quill.bubble.css';
@import '~react-quill/dist/quill.snow.css';
```
in react-components/index.scss, and now there is only `react-quill/dist/quill.snow.css` ?

Normal ?

Reply:
Yeah, core was just contained in snow, and bubble is a different theme that, as far as I could see, we were not using

Reply:
Fine, even lighter :)